### PR TITLE
DOCS-2369: Combine TrueCommand VM Deploy Instructions Into Main Insta…

### PR DESCRIPTION
**Pull request by jervin777**
…ll article

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

**Comment by Mrt134** [comment]
Looks good, pinging @beanpole135 for a quick review

**Comment by beanpole135** [comment]
Looks like the HTML within the boxes is not rendering on the preview site:
image

**Comment by beanpole135** [comment]
Looking at the code more closely, that `</div>` in the second box should be at the end of the first box instead.

File: content/TrueCommand/TCGettingStarted/Install/_index.md [hunk 1]
```diff
@@ -14,6 +14,70 @@ TrueCommand Cloud is the preferred method for using TrueCommand, as this option
 Click one of the tabs below to see instructions for your preferred deployment method.
 
 {{< tabs "TrueCommand Install Options" >}}
+{{< tab "VM Deployment" >}}
+{{< toc >}}
```
Comment (Mrt134) [inline content/TrueCommand/TCGettingStarted/Install/_index.md:18]:
```suggestion

```

File: content/TrueCommand/TCGettingStarted/Install/_index.md [hunk 2]
```diff
@@ -14,6 +14,70 @@ TrueCommand Cloud is the preferred method for using TrueCommand, as this option
...
+
+Deploying TrueCommand on a virtual machine (VM) requires different methods depending on what operating system you intend to use. 
+
+You can find VM images and setup instructions on our [TrueCommand-install](https://github.com/ixsystems/truecommand-install) GitHub repository.
+
+## Linux
+
+{{< tabs "Deploy TrueCommand on a Linux VM" >}}
+{{< tab "Debian" >}}
```
Comment (Mrt134) [inline content/TrueCommand/TCGettingStarted/Install/_index.md:27]:
```suggestion
### Debian
```

File: content/TrueCommand/TCGettingStarted/Install/_index.md [hunk 3]
```diff
@@ -14,6 +14,70 @@ TrueCommand Cloud is the preferred method for using TrueCommand, as this option
...
+If you don't already have it, you can download the VM image [here](https://www.debian.org/CD/http-ftp/).
+
+{{< hint info >}}
+Ensure you have the "wget" utility installed first: `apt-get install wget`
+{{< /hint >}}
+
+Run this command (as root) from a system terminal:  
+`wget https://raw.githubusercontent.com/iXsystems/truecommand-install/main/debian/setup.sh -O - | bash`
+{{< /tab >}}
+{{< tab "Alpine" >}}
```
Comment (Mrt134) [inline content/TrueCommand/TCGettingStarted/Install/_index.md:38]:
```suggestion

### Alpine
```

File: content/TrueCommand/TCGettingStarted/Install/_index.md [hunk 4]
```diff
@@ -14,6 +14,70 @@ TrueCommand Cloud is the preferred method for using TrueCommand, as this option
...
+Ensure you have the "wget" utility installed first: `apk add wget`.   
+Ensure that you have the "community" package repository enabled:
+* Edit the */etc/apk/repositories* file as root and uncomment the community repository line.
+* Run `apk update` to refresh the list of available packages.
+{{< /hint >}}
+
+Run this command (as root) from a system terminal:   
+`wget https://raw.githubusercontent.com/iXsystems/truecommand-install/main/alpine/setup.sh -O - | sh`
+{{< /tab >}}
+{{< tab "Void" >}}
```
Comment (Mrt134) [inline content/TrueCommand/TCGettingStarted/Install/_index.md:52]:
```suggestion

### Void
```

File: content/TrueCommand/TCGettingStarted/Install/_index.md [hunk 5]
```diff
@@ -14,6 +14,70 @@ TrueCommand Cloud is the preferred method for using TrueCommand, as this option
...
+If you don't already have it, you can download the VM image [here](https://voidlinux.org/download/).
+
+{{< hint info >}}
+Ensure you have the "wget" utility installed first: `xbps-install -y wget`
+{{< /hint >}}
+
+Run this command (as root) from a system terminal:   
+`wget https://raw.githubusercontent.com/iXsystems/truecommand-install/main/void/setup.sh -O - | bash`
+{{< /tab >}}
+{{< /tabs >}}
```
Comment (Mrt134) [inline content/TrueCommand/TCGettingStarted/Install/_index.md:63]:
```suggestion

```

**Review by Mrt134 — CHANGES_REQUESTED** [review]
Added a few suggestions to clean up the div errors @beanpole135 mentioned.

**Review by Mrt134 — APPROVED** [review]
Changes are addressed and this is ready for merge. I'm checking with Web team about setting up redirects or aliases for the vmdeploy URL that will be going offline before merging this one.

**Comment by Mrt134** [comment]
Opened https://jira.ixsystems.com/browse/DM-304 to notify the web team about the URL change and have approval to merge.

**Comment by Mrt134** [comment]
Link checker hit a temporary FreeBSD site outage - approved for merge.

**Pull request merged**